### ISSUE TITLE
PIP dependencies don't seem to list marshmallow.

### ISSUE THREAD

**Issue by dragonpaw** [marshmallow-code/django-rest-marshmallow#138]
Installing this module via poetry/pip didn't pull down marshmallow, so I think you're missing a dependency there.

**Comment by dragonpaw** [comment]
Looks like rest_framework is also missing there.

**Comment by sloria** [comment]
Indeed, this package declares no explicit dependencies in [`install_requires`](https://github.com/marshmallow-code/django-rest-marshmallow/blob/a7f6f4ee64135000db6352292c313112e5413be5/setup.py#L71), and that seems to have been an explicit decision from the beginning (correct me if I'm wrong @tomchristie ). My guess is that django and marshmallow are considered ["peer dependencies"](https://docs.npmjs.com/files/package.json#peerdependencies) that the consumer is responsible for installing themselves.

In Python packaging, though, everything is a peer dependency, so I think it would be fine to add both django and marshmallow as explicit dependencies in setup.py .

**Comment by dragonpaw** [comment]
I guess I'm of the opinion that once I install a package in Python, I should at least be able to import it without fatal errors. Optional dependencies are one thing, but if a package won't even import at all, that feels like the case for a explicit dependency to me.

(Especially as the docs for this package do mention specific versions you require.)

**Comment by sloria** [comment]
Ok, let’s go ahead and add them. Would you like to send a PR?
